src/user/helpers.js

```diff
@@ -0,0 +1,27 @@
+import { find } from "lodash";
+
+const userIds = [];
+
+export async function createUser(xo, params) {
+  const userId = await xo.call("user.create", params);
+  userIds.push(userId);
+  return userId;
+}
+
+export async function deleteAllUsers(xo) {
+  await Promise.all(
+    userIds.map(id =>
+      xo.call("user.delete", { id }).catch(error => console.error(error))
+    )
+  );
+  userIds.length = 0;
+}
+
+export function getAllUsers(xo) {
+  return xo.call("user.getAll");
+}
+
+export async function getUser(xo, id) {
+  const users = await getAllUsers(xo);
+  return find(users, { id });
+}
```

Comment:
Maybe create a class extending `Xo` with the methods you want.

Reply:
My bad, is a great idea. I thought that letting the methods locally to the user module will reduce the global code complexity. But, i'm wrong.